build/purger/purger.py

```diff
@@ -205,14 +202,19 @@ def wait_and_purge_room(room_id: str = None, event_id: str = None) -> None:
                 for i, row in enumerate(cur):
                     click.secho(f"{i}: {row}")
 
+        global_user_activity = dict()
+        global_user_activity["last_update"] = int(time.time()) - USER_PURGING_THRESHOLD - 1
+        global_user_activity["network_to_users"] = dict()
+
         # check if user activity file already exists
         if USER_ACTIVITY_PATH.exists():
-            global_user_activity = json.loads(USER_ACTIVITY_PATH.read_text())
+            try:
+                global_user_activity = json.loads(USER_ACTIVITY_PATH.read_text())
+            except JSONDecodeError:
+                click.secho(f"{USER_ACTIVITY_PATH} is not a valid JSON. Creating a new file.")
         else:
```

Comment:
This is not executing if the json is invalid, so the new file is not created. This should be something like:

```python
if global_user_activity is None:
```